smarts/sstudio/types.py

```diff
@@ -337,11 +343,20 @@ class TrapEntryTactic(EntryTactic):
 
 
 @dataclass(frozen=True)
-class Mission:
+class MissionBase:
```

Comment:
I had something in mind with this to add in all the shared attributes of the missions but the properties of frozen `dataclass` make it impossible to add default values on the base.

Reply:
I think I will strip out.